### PR TITLE
Correction for compilations

### PR DESCRIPTION
**Pull request by MathieuDutSik**
# Pull Request Summary
 
The PR addresses a few compilation errors that arise when some unfortunate combination of switches is selected, mostly at the debugging stage. Still I think it is important to have correct code that compiles.

## Description

Several entries corrected.

### Issue(s) addressed

No issue were raised, but the fact that some combination of switches do not compile had been the subject of complaints at te developer meeting.

### Check list  

- [ ] Branch is up to date with the authoritative repository (NOAA-EMC) develop branch. 
- [ ] Checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop). 
- [ ] If a version number update is required, checked the [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number) checklist. 
- [ ] If a new feature was added, a regression test for testing the new feature is added. 
 
### Testing

* How were these changes tested?
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
* Have the matrix regression tests been run (if yes, please note HPC and compiler)?
* Please indicate the expected changes in the regression test output, (Note the [list](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-develop#4-look-at-results) of known non-identical tests.)
* Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):

### CODE REVIEW

File: model/src/w3src3md.F90 [hunk 1]
```diff
@@ -1255,7 +1255,14 @@ SUBROUTINE W3SDS3 (A, K, CG, EMEAN, FMEAN, WNMEAN, USTAR, USDIR,    &
         DOUT(IK,ITH) = D(ITH+(IK-1)*NTH)
       END DO
     END DO
+<<<<<<< HEAD
```
Comment (MatthewMasarik-NOAA) [inline model/src/w3src3md.F90]: Hi @MathieuDutSik, could you resolve this conflict then I can start the review?

Comment (MathieuDutSik) [reply model/src/w3src3md.F90]: Sorry for that mistake

Comment (MathieuDutSik) [reply model/src/w3src3md.F90]: It has been corrected.

Comment (MatthewMasarik-NOAA) [reply model/src/w3src3md.F90]: No worries, thanks @MathieuDutSik!  I'll start reviewing.

**Review by MatthewMasarik-NOAA — APPROVED** [review]
The code changes all look good to me.  Matrix regression tests were run with hera.intel.  Only the known non-identical differences were found.

```bash
**********************************************************************  
********************* non-identical cases ****************************       
**********************************************************************    
mww3_test_03/./work_PR2_UNO_MPI_d2                     (12 files differ) 
mww3_test_03/./work_PR1_MPI_d2                     (22 files differ)         
mww3_test_03/./work_PR3_UNO_MPI_d2_c                     (12 files differ)
mww3_test_03/./work_PR3_UQ_MPI_d2_c                     (15 files differ)  
mww3_test_03/./work_PR3_UNO_MPI_d2                     (15 files differ) 
mww3_test_03/./work_PR2_UQ_MPI_d2                     (15 files differ) 
mww3_test_03/./work_PR3_UQ_MPI_d2                     (15 files differ) 
ww3_ta1/./work_UPD0F_U                     (0 files differ) 
ww3_tp2.10/./work_MPI_OMPH                     (6 files differ)  
ww3_tp2.16/./work_MPI_OMPH                     (4 files differ)  
ww3_tp2.6/./work_ST0                     (1 files differ) 
ww3_tp2.6/./work_ST4                     (1 files differ)   
ww3_ufs1.3/./work_a                     (1 files differ) 
**********************************************************************  
************************ identical cases ***************************** 
**********************************************************************
```
* [matrixCompSummary.txt](https://github.com/NOAA-EMC/WW3/files/9824545/matrixCompSummary.txt)
* [matrixCompFull.txt](https://github.com/NOAA-EMC/WW3/files/9824547/matrixCompFull.txt)
* [matrixDiff.txt](https://github.com/NOAA-EMC/WW3/files/9824546/matrixDiff.txt)
<hr>
 @MathieuDutSik, Thanks for your continued work in cleaning up the code!

**Pull request merged**